dags/plugins/operators_utils.py

```diff
@@ -0,0 +1,53 @@
+"""
+File: operators_utils.py
+Author: Paul-Emile Giacomelli
+Date: Fall 2023
+Description: Utils functions to be called by Airflow operators
+"""
+
+
+# Imports
+
+
+class OperatorsUtils:
```

Comment:
I probably would've chosen to keep them as functions for this use case, but lets try it out as a class and see how it works.

Reply:
Okay, I will leave the class for now.